### PR TITLE
Enable cors as an option. Docker default to allowing all.

### PR DESCRIPTION
**Pull request by andresuribe87**
With this change, you can request resources from a browser after running `mage run`

### CODE REVIEW

File: integration/common.go [hunk 1]
```diff
@@ -353,9 +353,16 @@ func getJSONElement(jsonString string, jsonPath string) (string, error) {
...
 	logrus.Println(fmt.Sprintf("\nPerforming GET request to:  %s\n", url))
 
-	resp, err := http.Get(url) // #nosec: testing only.
+	req, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		return "", errors.Wrapf(err, "getting url: %s", url)
+		return "", errors.Wrap(err, "building http req")
+	}
+
+	req.Header.Set("Origin", "http://foo.com")
```
Comment (nitro-neal) [inline integration/common.go:361]: did you mean to do this for every get() request?

**Review by nitro-neal — APPROVED** [review]
Awesome! Did you try to do a req from a react app or something for the 'real deal' test?

**Review by decentralgabe — APPROVED** [review]

File: integration/common.go (hunk 1 above)
Comment (andresuribe87) [reply integration/common.go]: Actually removed this. An actual test would need to check the presence of a header.

**Comment by andresuribe87** [comment]
> Awesome! Did you try to do a req from a react app or something for the 'real deal' test?

I did a request from the chrome dev tools in `localhost:12345`

**Pull request merged**